### PR TITLE
chore: fix integration and system tests

### PR DESCRIPTION
**Pull request by dkontorovskyy**
- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [ ] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does
Fix integration & system test:
- Java image no longer exists => renamed to openjdk
- Change internal URL and rotated credentials
- remove Jest callbacks

### Notes for the reviewer

_Instructions on how to run this locally, background context, what to review, questions…_

### Screenshots

_Visuals that may help the reviewer_

### CODE REVIEW

File: test/system/kind.spec.ts [hunk 1]
```diff
@@ -139,19 +123,15 @@ test('Kubernetes-Monitor with KinD', async (jestDoneCallback) => {
...
-            // also should have version here but due to test limitation it is undefined
-            // as it is injected as an environment variable via the Helm chart
-          });
-        } catch (error) {
-          jestDoneCallback(error);
-        }
+        expect(requestBody).toEqual<
+          Partial<transmitterTypes.IClusterMetadataPayload>
+        >({
+          agentId: expect.any(String),
```
Comment (kat1906) [inline test/system/kind.spec.ts:129]: why are we not checking the agentId matches the one given anymore?

Comment (dkontorovskyy) [reply test/system/kind.spec.ts:129]: I should revert that probably. Good catch

File: test/helpers/kubernetes-upstream.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ const UPSTREAM_POLLING_CONFIGURATION = {
 export async function getUpstreamResponseBody(
   relativeUrl: string,
 ): Promise<any> {
-  const url = `https://${config.INTERNAL_PROXY_CREDENTIALS}@kubernetes-upstream-int.dev.snyk.io/${relativeUrl}`;
+  const url = `https://${config.INTERNAL_PROXY_CREDENTIALS}@kubernetes-upstream-int${config.INTERNAL_PROXY_HOST}/${relativeUrl}`;
```
Comment (minsiyang) [inline test/helpers/kubernetes-upstream.ts:22]: what does the `config.INTERNAL_PROXY_HOST` look like? Would we need a preceding `.` for it?

Comment (dkontorovskyy) [reply test/helpers/kubernetes-upstream.ts:22]: Yes, it includes trailing `.` in the env variables :)

Comment (dkontorovskyy) [reply test/helpers/kubernetes-upstream.ts:22]: Should I remove and add it explicitly for easier understanding?

Comment (minsiyang) [reply test/helpers/kubernetes-upstream.ts:22]: maybe. It looks a bit confusing to include the `.`. 😅

Comment (dkontorovskyy) [reply test/helpers/kubernetes-upstream.ts]: Fixed :)

**Review by minsiyang — APPROVED** [review]

**Pull request merged**